features/step_definitions/step_definitions.rb

```diff
@@ -0,0 +1,20 @@
+When "I visit the {string} path" do |path|
+  visit ENV.fetch("TEST_URL", "") + path
```

Comment:
WIth the suggested change to the config, I think this can become
```suggestion
  visit path
```